core/src/main/java/org/frankframework/management/bus/endpoints/Monitoring.java

```diff
@@ -131,7 +129,9 @@ public Message<String> addMonitorOrTrigger(Message<?> message) {
 		try {
 			monitor.configure();
 		} catch (ConfigurationException e) {
-			throw new BusException("unable to (re)configure Monitor", e);
+			log.info("Unable to (re)configure monitor [{}]", monitor.getName(), e);
+			// Throw this as Warning / Bad Request, not able to configure was likely due to bad user input and not due to internal server error
```

Comment:
Is this something to be solved in the new PR you mentioned? Otherwise you might return something like `return new EmptyMessage(400);` here?